tests/core.py

```diff
@@ -678,6 +678,10 @@ def test_task_state(self):
             'task_state', 'example_bash_operator', 'runme_0',
             DEFAULT_DATE.isoformat()]))
 
+    def test_dag_state(self):
+        cli.dag_state(self.parser.parse_args([
+            'dag_state', 'example_bash_operator', DEFAULT_DATE.isoformat()]))
```

Comment:
I am not getting any pointers to add tests for this, can you give me some guidelines?